.github/workflows/docker-publish.yaml

```diff
@@ -47,12 +47,12 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      # Build And Push Image
-      - name: Build Docker image
-        run: docker compose -f docker-compose-internal.yaml build
-      - name: Push Docker image
+      # Build And Push Images
+      - name: Build Docker images
+        run: docker buildx bake
+      - name: Push Docker images
         if: github.ref == 'refs/heads/master'
-        run: docker compose -f docker-compose-internal.yaml push
```

Comment:
What do you think of making it a separate workflow and then deprecating this one?

Reply:
Feel it's not needed. If ever we have a problem with this we can git revert the PR.